### ISSUE TITLE
Data entry - Seasons tab - Features - Description fields

### ISSUE THREAD

**Issue by RobinaSanderson** [AtlasOfLivingAustralia/seasonal-calendar#58]
Change description field  label to "Brief description"

Create new field with label "More" 
This field should accept up to 10 000 characters

If not part of the initial implementation this will need to be added to the web service passing information to be stored in Profiles.

This should be done prior to visualisation work.
